src/Qmmands/Builders/CommandBuilder.cs

```diff
@@ -97,7 +97,14 @@ internal CommandBuilder(ModuleBuilder module, object callback)
             Module = module;
             Callback = callback;
             Cooldowns = new List<Cooldown>();
+
+            #if NETSTANDARD2_0
+            Aliases = new HashSet<string>();
```

Comment:
This change is not a net positive. It causes the set to be created with whatever defaults are there, which might cause growing and reallocations. Standard 2.0 should very much support initial capacity argument.

Reply:
Standard 2.0 does not support that argument, unfortunately.